### PR TITLE
Payment privacy page minor Lightning iteration

### PR DESCRIPTION
**Pull request by GBKS**
Addresses #575.

This is meant as a minimal revision to this page to add Lightning-related privacy information. I am not the best-versed on privacy, so a good review process would be helpful.

We should also clarify the scope of this page. Is it to explain the privacy pitfalls to designers? Or to give specific design recommendations and guidelines. If it's the latter, shouldn't these things already be covered in the send and receive pages that already show full UI mock-ups and flows?

🔒[Check out the preview](https://deploy-preview-636--sad-borg-390916.netlify.app/guide/payments/privacy/)🕵️‍♀️

### CODE REVIEW

**Review by pavlenex — APPROVED** [review]
LGTM

**Comment by GBKS** [comment]
Had to resolve a conflict. Would appreciate another review, por favor.

File: guide/payments/privacy.md [hunk 1]
```diff
@@ -34,35 +34,45 @@ This page should inform about what information is made public when sending or re
...
 -->
 
-It’s a common misconception that Bitcoin payments are anonymous. Instead, they can be referred to as pseudonymous, this means that who owns a freshly generated addresses is not public knowledge. Unless your ownership is revealed, whether directly by yourself or indirectly by some third-party you are able to remain anonymous.
+It’s a common misconception that Bitcoin payments are anonymous. Rather, bitcoin payments are [pseudonymous](https://en.wikipedia.org/wiki/Pseudonym), meaning no identifiable information is tied to transactions. Unless ownership is revealed, whether by the parties involved or some third-party, payments remain anonymous.
 
-Transactions, their signatures, and addresses added to the Bitcoin blockchain remain public forever. This means that looking up any address or transaction is trivial, as demonstrated by going back to the very first block mined on [January 3, 2009](https://blockstream.info/tx/4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b).
+It is important to differentiate between privacy on the Bitcoin network and Lightning network. Their differences in technical architecture results in different privacy considerations.
```
Comment (pavlenex) [inline guide/payments/privacy.md:39]:
```suggestion
It is important to differentiate between Bitcoin and Lightning network privacy. Their differences in technical architecture result in different privacy considerations.
```

**Review by pavlenex — COMMENTED** [review]
Just a minor re-wording. This LGTM, the scope of this page may be expanded later on, if a contributor passionate about privacy decides to dedicate time to it, but this, at least to me, looks like a good high-level overview that should be enough for most designers.

**Review by pavlenex — APPROVED** [review]

**Pull request merged**